Scenes/HUD/ElementTowersMenu.gd

```diff
@@ -58,6 +59,9 @@ func _ready():
 	_on_element_changed()
 
 
+func _process(_delta):
```

Comment:
Change this to a connection to button's "toggled()" signal.

In general, logic like this should be done using signals instead of _process(), if it's possible.

Reply:
Done